src/Polkadot/Extrinsic.cpp

```diff
@@ -208,6 +211,32 @@ Data Extrinsic::encodeStakingCall(const Proto::Staking& staking, TWSS58AddressTy
             append(data, getCallIndex(network, stakingChill));
             break;
 
+        case Proto::Staking::kChillAndUnbond:
```

Comment:
Have you sent this tx to network test?

Reply:
not yet, figuring out how to run the console to test it

Reply:
wallet console doesn't have this feature, currently we have to manually send it